packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/index.tsx

```diff
@@ -238,7 +238,12 @@ const CompleteResetPassword = ({
                 `/signin_totp_code${location.search}`
               );
               isHardNavigate = true;
-            } else if (sessionIsVerified && isOAuthIntegration(integration)) {
+            } else if (
```

Comment:
This also might not be needed. This block is mainly here to redirect back to the RP, which we won't do after #15820 lands.

Reply:
Yep 👍